### ISSUE TITLE
Incorrect detection

### ISSUE THREAD

**Issue by Kr0sshair** [restonic4/OreDetector#1]
It just does its beep beep beep and i go under and theres no diamonds, i came to the deep dark, ancient city and bedrock and it didnt reveal anything. waste of my diamonds.

**Comment by restonic4** [comment]
Hmm, thats weird, what version are you playing on? on what loader? are you sure there is no diamonds nearby?

**Comment by Kr0sshair** [comment]
0 diamonds, latest fabric on 1.20.4. might world gen mods cause an issue with distribution of ores being different

**Comment by Kr0sshair** [comment]
Happened twice... Once i came to the deep dark the other one i went under lava n shit, no diamonds.

**Comment by Kr0sshair** [comment]
Oops forgot to mention, once a random point passes [DOWN THE SAME COLLUMN X/Z  COORDINATE] it stops doing its beep beep beep. Like its "yo trust me theres diamonds there" "really?" "Nuh uh i lied"

**Comment by restonic4** [comment]
> Oops forgot to mention, once a random point passes [DOWN THE SAME COLLUMN X/Z  COORDINATE] it stops doing its beep beep beep. Like its "yo trust me theres diamonds there" "really?" "Nuh uh i lied"

weird, can you send me your installation and world save? so i can replicate it and fix it.

do you have discord?

**Comment by Kr0sshair** [comment]
Yeah i got discord. Ill get home at around 3 pmtoday, its 9:43 rn. Ill give you me and my sisters survival mode. Though the modpack is quite big, it happens with only ore detector as well. ".krosshair" on discord.

**Comment by Kr0sshair** [comment]
Uh helloo?? Ill be home soon, add me on discord -> [ .krosshair ]
